client/src/components/Home/AboutUs/AboutUs.css

```diff
@@ -1,190 +1,194 @@
-    width: 50%;
-    border: 2px solid rgb(255, 255, 255);
-    border-radius: 20px;
-    top:63%;
-    left: 20%;
-    box-shadow: 0 6px 10px rgba(0,0,0,.08), 0 0 6px rgba(0,0,0,.05) !important;
-    cursor: pointer;
-    z-index: 20;
-  
-}
-
-.about-us-section .image1:hover,.about-us-section .image2:hover,.about-us-section .image3:hover,.about-us-section .image4:hover{
-    transform: scale(1.05);
-    transition: transform 0.3s ease;
-    box-shadow: 5px 10px 20px 1px rgba(0,0,0,0.253) !important;
-    z-index: 999;
-}
-.modalimage1 {
-    width: 100%;
-    height: 100%;
-}
-
-/* @media (max-width:1200px){
-    .about-us-section .image1,.about-us-section .image2,.about-us-section .image3,.about-us-section .image4{
-        position: relative;
-        top: 0%;
-        left: 0%;
-        width:50%;
-        height:12vw;
-        margin-top: 3%;
-        display: block;
-        margin-left: auto;
-        margin-right: auto;
-    }
-} */
-
-
-/* @media (max-width:992px){
-
-    .about-us-section .paragraph{
-        text-align: center;
-    }
-
-    .about-us-section .image1,.about-us-section .image2,.about-us-section .image3,.about-us-section .image4{
-        position: relative;
-        top: 0%;
-        left: 0%;
-        width:30vw;
-        height:20vw;
-        margin-top: 3%;
-        margin-left: 3px;
-        margin-right: 3px;
-        
-    }
-    .image-row{
-        overflow: scroll;
-        white-space: nowrap;
-    }
-    .about-us-section .paragraph{
-        font-size: 15px;
-        padding-left: 20px;
-        padding-right: 20px;
-        
-    }
-}
-@media (max-width:768px){
-
-
-    .about-us-section .image1,.about-us-section .image2,.about-us-section .image3,.about-us-section .image4{
-        width:45vw;
-        height:35vw;
-        
-    }
-        
-    
-} */
-
-.about-us-section {
-    font-family: 'Raleway', sans-serif;
-    color: white;
-    border: 1px solid #EE2A7B;
-    /* box-sizing: border-box; */
-    border-radius: 23px;
-    width: 85vw;
-}
-
-.about-us-section .title {
-    color: #EE2A7B;
-    font-size: 6vh;
-    margin-left: 8vw;
-    margin-bottom: 5vh;
-    margin-top: 8vh;
-}
-
-.about-us-section .content {
-    color: white;
-    font-size: 3.8vh;
-    margin-left: 8vw;
-    margin-right: 8vw;
-    margin-bottom: 5vh;
-}
-.about-us-section #about-us {
-    color: #1EABD6;
-    font-size: 1.5vw;
-    position: relative;
-    left:47vw;
-    bottom: 20vh;
-    transform: rotate(-90deg);
-    height:0;
-}
-
-@media screen and (max-width: 426px) {
-    .about-us-section {
-        border: none;
-    }
-
-    .about-us-section .title {
-        margin-left: 0;
-    }
-
-    .about-us-section .content {
-        color: white;
-        font-size: 2.5vh;
-        margin-left: 0;
-        margin-right: 0;
-        margin-bottom: 5vh;
-    }
-
-    .about-us-section #about-us {
-        height: 0;
-        opacity: 0;
-    }
-}
\ No newline at end of file
+/* .about-us-section .heading{

```

Comment:
Are these intentionally commented?
If the commented code is not required, you can go ahead and delete it?
@EmperorYP7 You can have a look here!

Reply:
It was commented out by @EmperorYP7 so I didn't make any changes to that.
@EmperorYP7 is it required or shall I delete that?

Reply:
my bad! After the @EmperorYP7 comment, I will merge it! :tada:

Reply:
Looks like I forgot to clean this up. Go ahead and delete this commented section out @hariom1625.